lib/leg_by_zip.rb

```diff
@@ -0,0 +1,7 @@
+class LegByZip
```

Comment:
Later, if you end up searching for Legislators by city, or state, or zip and city, or zip, state, city, and gender, this class name would be an issue. A better approach is to name this: `LegislatorSearcher` and have a `by_zipcode` method, and one for states and zip, etc etc. Just food for thought.